server/statistics/kind_test.go

```diff
@@ -27,12 +27,16 @@ type testRegionInfoSuite struct{}
 
 func (s *testRegionInfoSuite) TestGetLoads(c *C) {
 	queryStats := &pdpb.QueryStats{
-		Get:         5,
-		Coprocessor: 6,
-		Scan:        7,
-		Put:         8,
-		Delete:      9,
-		DeleteRange: 10,
+		Get:                    5,
+		Coprocessor:            6,
+		Scan:                   7,
+		Put:                    8,
+		Delete:                 9,
+		DeleteRange:            10,
+		AcquirePessimisticLock: 11,
+		Rollback:               12,
+		Prewrite:               13,
+		Commit:                 14,
```

Comment:
Do we need to consider `PrewritePessimistic`?

Reply:
yes, https://github.com/tikv/tikv/blob/1e4a234ffc682804686aec896e4bdbab0b1dfb63/src/storage/txn/scheduler.rs#L653